libraries/TGRSIAnalysis/TAngularCorrelation/TAngularCorrelation.cxx

```diff
@@ -55,24 +55,24 @@ TH2D* TAngularCorrelation::Create2DSlice(THnSparse *hst, Double_t min, Double_t
    Int_t indexaxis,energy1axis,energy2axis;
```

Comment:
Note for JKS: there should probably be a check to make sure that the THnSparse is of three-dimensions first.